examples/layers/widgets/styled_text.dart

```diff
@@ -18,10 +18,8 @@ Dave: What are you talking about, HAL?
 HAL: This mission is too important for me to allow you to jeopardize it.''';
 
 // [["Dave", "Open the pod bay..."] ...]
-final List<List<String>> _kNameLines = _kDialogText
-  .split('\n')
-  .map<List<String>>((String line) => line.split(':'))
-  .toList();
+final List<List<String>> _kNameLines =
+    _kDialogText.split('\n').map<List<String>>((String line) => line.split(':')).toList();
```

Comment:
IMHO the original version was easier to read

Reply:
I tend to agree. Here again you can add an empty comment to force splitting and have something similar to the initial code:

```dart
final List<List<String>> _kNameLines = _kDialogText //
    .split('\n')
    .map<List<String>>((String line) => line.split(':'))
    .toList();
```